src/bot.py

```diff
@@ -43,8 +44,19 @@ def run(self) -> None:
         """Runs the bot"""
         if TOKEN is None:
             raise EnvironmentError("Token value not found in .env")
+        self.load_yaml()
         self.load_cogs()
         super().run(TOKEN)
 
+    def load_yaml(self) -> None:
+        """Loads Yaml files"""
+        logging.info("Loading YAML files...")
+
+        #with open("src/sources/trusted_seed_form.yml") as file:
```

Comment:
TODO: remove the comments, when merging PR from https://github.com/Vyvy-vi/TEC-Discord-Automation/tree/feat/trusted-seed-form